Makefile

```diff
@@ -14,30 +14,24 @@
 
 
 # Base path used to install.
-PLUGIN_DESTDIR ?= /opt/rsnapshotd
 CMD_DESTDIR ?= /usr/local
 GO111MODULE_VALUE=off
 PREFIX ?= out/
 
-PLUGINS=stargzfs-linux-amd64.so
-CMD=rsnapshotd ctr-remote
+CMD=ssnapshotd ctr-remote
```

Comment:
`containerd-stargz-grpc` may look more descriptive

https://github.com/AkihiroSuda/containerd-fuse-overlayfs/tree/master/cmd/containerd-fuse-overlayfs-grpc

Reply:
Thanks for the suggestion, I renamed the daemon. PTAL 🙏